test/test.oauth2.ts

```diff
@@ -1031,7 +1034,7 @@ it('getToken should set client_id if not provided in options', async () => {
   scope.done();
   assert(res.res);
   if (!res.res) return;
-  const params = qs.parse(res.res.config.data);
+  const params = qs.parse(res.res.config.body);
```

Comment:
These should all be data!

Reply:
I'm not sure! `data` wasn't there, but `body` was!

Reply:
1.1.1 just shipped - can you try that?

Reply:
It's there now! PR updated.